### DISCUSSION TITLE
Request: Update Renaitre Skin

### DISCUSSION THREAD

**Discussion by doums** [SFX-WoW/Masque#215] [Issues] [answered]
Hi,

It would be great to update [Renaitre](https://www.curseforge.com/wow/addons/masque-renaitre) skin.
I use it (circle fade version) as my main skin for Masque, and I think lot of people use it as well.

Thanks !

**Comment by StormFX** [comment, marked as answer]
This isn't a Masque issue. You need to use the project page of the skin to request updates.
